R/expect_comparison_linter.R

```diff
@@ -49,7 +49,7 @@
 #' @export
 expect_comparison_linter <- function() {
   # != doesn't have a clean replacement
-  comparator_nodes <- setdiff(as.list(infix_metadata$xml_tag[infix_metadata$comparator]), "NE")
```

Comment:
What's with this diff?

Reply:
I think it's a vestige from before `xp_or()` supported vector input (585e7cd3e0e4a3aa8f78fdf0242a908ff4cfd65e0). And `setdiff(list(), "NE")` looks weird to me.